test/ffmpeg-qsv/util.py

```diff
@@ -49,18 +49,34 @@ def have_ffmpeg_vp9_qsv_decode():
 def have_ffmpeg_h264_qsv_encode():
   return try_call("ffmpeg -hide_banner -encoders | grep h264_qsv")
 
+@memoize
+def have_ffmpeg_x264_encode():
+  return try_call("ffmpeg -hide_banner -encoders | grep libx264")
+
 @memoize
 def have_ffmpeg_hevc_qsv_encode():
   return try_call("ffmpeg -hide_banner -encoders | grep hevc_qsv")
 
+@memoize
+def have_ffmpeg_x265_encode():
+  return try_call("ffmpeg -hide_banner -encoders | grep libx265")
+
 @memoize
 def have_ffmpeg_mjpeg_qsv_encode():
   return try_call("ffmpeg -hide_banner -encoders | grep mjpeg_qsv")
 
+@memoize
+def have_ffmpeg_mjpeg_encode():
+  return try_call("ffmpeg -hide_banner -encoders | grep mjpeg")
```

Comment:
This will match mjpeg, mjpeg_qsv and mjpeg_vaapi.  Is there a good way to only match mjpeg?

Perhaps:
 ```
ffmpeg -hide_banner -encoders | awk '{print $2}' | grep mjpeg$
```
... could work.  Or something more robust?

Reply:
ffmpeg -hide_banner -encoders | awk '{print $2}' | grep -w mjpeg  -> added like this